factories/chapters.rb

```diff
@@ -3,6 +3,7 @@
 FactoryGirl.define do
   factory :chapter do
     content "Awesome content!"
+    title "Cool chapter title"
```

Comment:
Is it necessary to have a title for all chapters we test, or can we just set the title in the specific tests that need it?

Reply:
I tried setting it just in those tests, and couldn't get it to work that way.

Reply:
Will fix.